### PR TITLE
May k3s releases

### PR DESCRIPTION
**Pull request by briandowns**
https://github.com/rancher/rancher/issues/41615

### CODE REVIEW

**Review by rancher-max — CHANGES_REQUESTED** [review]
As this will be the first v1.26 release, I believe we want to replace v1.26.4. cc @snasovich @kinarashah to confirm.

Also, for 1.26, I noticed that we have some changed args for `flannel-backend`. I believe we need to update this to include `wireguard-native`.
`   --flannel-backend value                    (networking) backend<=option1=val1,option2=val2> where backend is one of 'none', 'vxlan', 'ipsec' (deprecated), 'host-gw', 'wireguard-native', 'wireguard' (deprecated) (default: "vxlan")`

**Review by rancher-max — COMMENTED** [review]
LGTM! Will officially approve once we cut releases and remove RC. Thank you!

File: channels.yaml
Comment (brandond) [inline channels.yaml:402]:
```suggestion
        - wireguard-native
```

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by brandond — DISMISSED** [review]
need to `go generate` again, LGTM otherwise.

**Review by rancher-max — APPROVED** [review]

File: channels.yaml
Comment (brandond) [reply channels.yaml:402]: still misspelled

**Pull request merged**
